pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1106,22 +1114,23 @@ type ThanosSpec struct {
 
 	// Defines the tracing configuration for the Thanos sidecar.
 	//
+	// `tracingConfigFile` takes precedence over this field.
+	//
 	// More info: https://thanos.io/tip/thanos/tracing.md/
 	//
-	// This is an experimental feature, it may change in any upcoming release
+	// This is an *experimental feature*, it may change in any upcoming release
```

Comment:
We mention tracing config as experimental because they are experimental features in Prometheus itself, right?

I wonder if we want to mention this specifically or not (this being experimental in Prometheus)

Reply:
> We mention tracing config as experimental because they are experimental features in Prometheus itself, right?

correct though it's for Thanos sidecar.

> I wonder if we want to mention this specifically or not (this being experimental in Prometheus)

I'd rather keep it simple and don't go into the details.